src/main/java/teammates/common/util/Const.java

```diff
@@ -528,6 +528,11 @@ private Const() {
                 "The weights for the choices of each Sub-question of a "
                 + Const.FeedbackQuestionTypeNames.RUBRIC
                 + " must be valid numbers with precision up to 2 decimal places.";
+
+        // Text Question
+        public static final int TEXT_MIN_RECOMMENDED_LENGTH = 0;
+        public static final String TEXT_ERROR_INVALID_RECOMMENDED_LENGTH = "Recommended length must be"
+                + " a non-negative integer";
```

Comment:
`Recommended length must be 0 or greater` might be more user friendly :smile:

Reply:
Got it 👍